test/jdk/javax/swing/JInternalFrame/6726866/bug6726866.java

```diff
@@ -36,27 +39,42 @@
 import javax.swing.JFrame;
 import javax.swing.JInternalFrame;
 import javax.swing.JLabel;
+import javax.swing.SwingUtilities;
+
+public class bug6726866 {
 
-public class bug6726866 extends JApplet {
+    private static final String instructionsText = """
+            Drag the internal frame inside the green undecorated window,
+            if you can drag it the test passes, otherwise fails. """;
 
-    public void init() {
-        JFrame frame = new JFrame("bug6726866");
-        frame.setUndecorated(true);
-        setWindowNonOpaque(frame);
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame passFailJFrame = new PassFailJFrame.Builder()
```

Comment:
Please use the `builder` helper method instead of using `new` explicitly.

Reply:
updated...